### PR TITLE
PCSUP-22237

### PR DESCRIPTION
**Pull request by jenjoe22**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--prisma-cloud-docs--hlxsites.hlx.page/
- After: https://<branch>--prisma-cloud-docs--hlxsites.hlx.page/
- Worker: https://prisma-cloud-docs-production.adobeaem.workers.dev/?branch=<branch>

### CODE REVIEW

File: docs/en/enterprise-edition/content-collections/dashboards/dashboards-get-started.adoc
[the remark this answers is not in the document]
Comment (jenjoe22) [reply docs/en/enterprise-edition/content-collections/dashboards/dashboards-get-started.adoc:1]: Sounds good. I'll update with your edits.

**Review by arane26 — COMMENTED** [review]
@jenjoe22 - Replace "right" with correct
When you set up Prisma Cloud Compute to secure your cloud workloads, ensure you've granted Prisma Cloud the correct permissions.

File: docs/en/classic/compute-admin-guide/configure/permissions.adoc [hunk 1]
```diff
@@ -1,18 +1,18 @@
 [#permissions]
 == Permissions by Feature
 
-When you set up Prisma Cloud Compute to secure your cloud workloads, you'll need to ensure you've granted Prisma Cloud the right permissions.
+When you set up Prisma Cloud Compute to secure your cloud workloads, ensure you've granted Prisma Cloud the right permissions.
 The following tables list the permissions required for each of Compute's protection capabilities.
 
 Review the following sets of permissions to ensure Prisma Cloud has the right permissions.
```
Comment (arane26) [inline docs/en/classic/compute-admin-guide/configure/permissions.adoc:7]: Delete 
Review the following sets of permissions to ensure Prisma Cloud has the right permissions.

File: docs/en/classic/compute-admin-guide/configure/permissions.adoc [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 
-* The needed for each cloud service provider:
+* Required for each cloud service provider:
```
Comment (arane26) [inline docs/en/classic/compute-admin-guide/configure/permissions.adoc:9]: Permissions required ...

**Pull request merged**